src/app/modules/gb-analysis-module/gb-analysis.component.css

```diff
@@ -1,6 +1,6 @@
 /* top, right, bottom, left */
 .analysis-container {
-  margin-top: var(--gb-navbar-height-extra);
+  margin-top: 52px; /* var(--gb-navbar-height-extra); */
```

Comment:
we need variable for this, for clean code

Reply:
I agree. I want to further investigate if we can make something like postcss do this automatically, or use scss instead. The preferred way is indeed to make use of all the great features of the latest languages and transpile for backward compatibility.